content/en/blog/_posts/2024-03-08-mid-cycle-1.30.md

```diff
@@ -0,0 +1,66 @@
+---
+layout: blog
+title: 'A Peek at Kubernetes v1.30'
+date: 2024-03-08
+slug: kubernetes-1-30-upcoming-changes
+---
+
+**Authors:** Amit Dsouza, Frederick Kautz, Kristin Martin, Abigail McCarthy, Natali Vlatko 
+
+## A quick look: exciting changes in Kubernetes v1.30
+
+It's a new year, and a new Kubernetes release. We're currently halfway through the release cycle and we have quite a few interesting and exciting enhancements coming with v1.30. From brand new features in alpha, to established features graduating to stable, to long-awaited improvements, this release has something for everyone to pay attention to!
+
+To tide you over until the official release, here's a sneak peak of the enhancements we're most excited about this cycle!
+
+## Major changes for Kubernetes v1.30
+
+### Structured parameters for dynamic resource allocation ([KEP-4381](https://kep.k8s.io/4381))
+
+[Dynamic resource allocation](/docs/concepts/scheduling-eviction/dynamic-resource-allocation/) was added to Kubernetes as an alpha feature in v1.26. It defines an alternative to the traditional device-plugin API for requesting access to third-party resources. By design, dynamic resource allocation uses parameters for resources that are completely opaque to core Kubernetes. This approach poses a problem for the Cluster Autoscaler (CA) or for any higher level controller that needs to make decisions for a group of pods (e.g. a job scheduler). It cannot simulate the effect of allocating or deallocating claims over time. Only the third-party DRA drivers have the information available to do this.
+
+​​Structured Parameters for dynamic resource allocation an extension to the original implementation that addresses this problem by building a framework to support making these claim parameters less opaque. Instead of handling the semantics of all claim parameters themselves, drivers could manage resources and describe them using a specific "structured model" pre-defined by Kubernetes. This would allow components aware of this "structured model" to make decisions about these resources without outsourcing them to some third-party controller. For example, the scheduler would be able to allocate claims rapidly, without back-and-forth communication with dynamic resource allocation drivers. Work done for this release centers on defining the framework necessary to enable different "structured models" and to implement the "named resources" model. This model allows listing individual resource instances and, compared to the traditional device plugin API, adds the ability to select those instances individually via attributes.
```

Comment:
```suggestion
​​Structured Parameters for dynamic resource allocation is an extension to the original implementation that addresses this problem by building a framework to support making these claim parameters less opaque. Instead of handling the semantics of all claim parameters themselves, drivers could manage resources and describe them using a specific "structured model" pre-defined by Kubernetes. This would allow components aware of this "structured model" to make decisions about these resources without outsourcing them to some third-party controller. For example, the scheduler would be able to allocate claims rapidly, without back-and-forth communication with dynamic resource allocation drivers. Work done for this release centers on defining the framework necessary to enable different "structured models" and to implement the "named resources" model. This model allows listing individual resource instances and, compared to the traditional device plugin API, adds the ability to select those instances individually via attributes.
```

The comment about line wrapping applies here.

Except for the missing "is" this looks good to me.

Now I just need to get https://github.com/kubernetes/kubernetes/pull/123516/ actually merged... exception request is pending.